### PR TITLE
Add pool column to new machine listing.

### PR DESCRIPTION
**Pull request by squidsoup**
## Done
* Add pool column to new machine listing.

## QA
* Ensure poll name and machine note are displayed correctly in new machine listing.

### CODE REVIEW

File: ui/src/app/machines/views/MachineList/PowerColumn/PowerColumn.js [hunk 1]
```diff
@@ -33,4 +34,8 @@ const PowerColumn = ({ systemId }) => {
   );
 };
```
Comment (squidsoup) [inline ui/src/app/machines/views/MachineList/PowerColumn/PowerColumn.js]: Driveby proptype fix.

File: ui/src/app/machines/views/MachineList/PoolColumn/PoolColumn.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <a className="p-link--soft" href="#/pools" title={machine.pool.name}>
+            {machine.pool.name}
+          </a>
+        </span>
+      </div>
+      <div
+        className="p-double-row__secondary-row u-truncate-text"
+        aria-label="Note"
+      >
+        <span title={machine.power_type} data-test="note">
```
Comment (Caleb-Ellis) [inline ui/src/app/machines/views/MachineList/PoolColumn/PoolColumn.js]: title should be `machine.description`
